### DISCUSSION TITLE
Elevate Winter map theme (OpenAndroMaps)

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#676] [Q&A] [answered]
The Elevate Winter Theme is available on the FTP server. Will you include it in the download table for OAM?

**Comment by devemux86** [comment]
Elevate Winter map theme will be available in the download manager of the next Cruiser.

https://www.senotto.de/Tipps_Tricks/GPS/OAM_Winter/OAM_Winter.html

<img src="https://github.com/user-attachments/assets/e48f4355-846f-40b1-875e-a1a43994cb9b" width="50%"/><img src="https://github.com/user-attachments/assets/6b4923dd-9437-41d3-a42f-ba30dfa9137a" width="50%"/>

**Reply by kds112** [reply]
Great! Will you announce it in the OAM forum, too? ⛷️❄️

**Reply by devemux86** [reply]
> Will you announce it in the OAM forum, too? ⛷️❄️

https://www.openandromaps.org/oam-forums/topic/elevate-winter-fork/page/24#post-56959

**Reply by devemux86** [reply]
Elevate Winter map theme is available in **[Cruiser 5.1.0-b12](https://github.com/devemux86/cruiser/discussions/671)**.

**Reply by MotoUKRider** [reply]
Nice to have some different styles/themes, but more layers than a Mary Berry jam-sponge :cake:  Glad I don't have to maintain *that* :clown_face:

**Reply by kds112** [reply]
This morning on OAM there was a Elevate Winter available dated 1st December. My installed version was 30th November, but no update was offered in Cruiser. Please have a look to the update routine...

**Reply by devemux86** [reply]
@kds112 Updates are notified after 1 day, please check again tomorrow.

**Comment by devemux86** [comment, marked as answer]
- Elevate Winter map theme is available in the download manager of https://github.com/devemux86/cruiser/discussions/684
